src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Better Bayesian Workfllows with Inference Data and Xarray'
+date: '2022-12-17'
```

Comment:
TODO: update date before merging

Reply:
@aloctavodia @OriolAbril want to add your names in the lines just below here?